front/app/api/campaign_consents/useUpdateCampaignConsents.ts

```diff
@@ -14,14 +14,13 @@ const updateCampaignConsents = async ({
       const idPart = campaignConsentId
         ? campaignConsentId
         : `by_campaign_id/${campaignId}`;
-      const tokenPart =
-        typeof unsubscriptionToken === 'string'
-          ? `?unsubscription_token=${unsubscriptionToken}`
-          : '';
       return fetcher<ICampaignConsent>({
-        path: `/consents/${idPart}${tokenPart}`,
+        path: `/consents/${idPart}`,
         action: 'patch',
-        body: { consent: { consented } },
+        body: {
+          consent: { consented },
+          unsubscription_token: unsubscriptionToken,
+        },
```

Comment:
Following the docs: https://developers.citizenlab.co/api-docs/ee/frontweb_api/master/campaign_consents/update_a_campaign_consent_by_campaign_id_using_an_unsubscription_token.html